link-packages.sh

```diff
@@ -0,0 +1,2 @@
+#/bin/sh
+ln -f -s /usr/lib/python2.7/dist-packages/macaroons.so devenv/lib/python2.7/site-packages
```

Comment:
If this is only used in the Makefile, I wonder if it should just be a makefile target and not a separate script. [shrug]